css/templates/lune/components/_sidebar.css

```diff
@@ -0,0 +1,99 @@
+@import "../page-vars";
+
+.ampstart-sidebar {
+  width: 80vw;
+  /* Keep the header bar visible when the sidebar is open */
+  height: calc(100vh - var(--layout-header-height-mobile));
+  margin-top: var(--layout-header-height-mobile);
+  padding-bottom: calc(var(--layout-header-height-mobile) / 2);
+
+  display: flex;
```

Comment:
It could work once #499 and https://github.com/ampproject/amphtml/issues/10380 are fixed, but since we don't want to slow this PR down, we preferred to amend the design. We may revisit the sidebar later on, once these issues are solved.

Re the sidebar animation, it does animate upon being closed, but the animation is not as smooth as when it's opened
<img width="317" alt="screen shot 2017-07-14 at 10 41 12" src="https://user-images.githubusercontent.com/1083581/28207284-10fab40e-6881-11e7-9791-15711b6d96b0.png">
.